nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -1,6 +1,5 @@
 collision_monitor:
   ros__parameters:
-    use_sim_time: True
```

Comment:
This is the only critical change from what I can see as it addresses our VSS. Good thing is we already bubbled down/up the `use_sim_time` parameter anyway ([here](https://github.com/botsandus/auto-robot/blob/525831cb4173d2fcb9940a9a0030917299c4a258/virtual_safety_system/launch/arri_collision_monitor.launch.py#L42)) and also use a cusomt yml file ([here](https://github.com/botsandus/auto-robot/blob/525831cb4173d2fcb9940a9a0030917299c4a258/virtual_safety_system/launch/arri_collision_monitor.launch.py#L34))